clang/test/SemaCXX/builtin-std-move.cpp

```diff
@@ -130,27 +137,24 @@ C &&(&rMove)(C&) = std::move; // #8 expected-note {{instantiation of}}
 C &&(&rForward)(C&) = std::forward<C>; // #9 expected-note {{instantiation of}}
 int (&rUnrelatedMove)(B, B) = std::move;
 
-#if __cplusplus <= 201703L
-// expected-warning@#1 {{non-addressable}}
-// expected-warning@#2 {{non-addressable}}
-// expected-warning@#3 {{non-addressable}}
-// expected-warning@#4 {{non-addressable}}
-// expected-warning@#5 {{non-addressable}}
-// expected-warning@#6 {{non-addressable}}
-// expected-warning@#7 {{non-addressable}}
-// expected-warning@#8 {{non-addressable}}
-// expected-warning@#9 {{non-addressable}}
-#else
-// expected-error@#1 {{non-addressable}}
-// expected-error@#2 {{non-addressable}}
-// expected-error@#3 {{non-addressable}}
-// expected-error@#4 {{non-addressable}}
-// expected-error@#5 {{non-addressable}}
-// expected-error@#6 {{non-addressable}}
-// expected-error@#7 {{non-addressable}}
-// expected-error@#8 {{non-addressable}}
-// expected-error@#9 {{non-addressable}}
-#endif
+// cxx17-warning@#1 {{non-addressable}}
```

Comment:
I might suggest 'collating' these a bit, that is, put all of the ones for the same line number 'next' to eachother, rather than by language mode.  But I don't feel too strongly.

Reply:
I'd put them directly after the lines they are issues for, instead of gathering them in one place and asking readers to follow the markers. Then they will be naturally collated by line instead of language mode.

Reply:
Even better!

Reply:
I’ve put them on the lines that cause the diagnostics and also aligned them because I found it to be too unreadable otherwise.